Makefile

```diff
@@ -50,10 +91,50 @@ behat.yml: behat.yml.dist
 		cp behat.yml.dist behat.yml;\
 	fi
 
-book:
-	#
-	# Darkmira Community Event Manager
-	#
-	# Application: http://0.0.0.0:8080
-	# Minio:       http://0.0.0.0:9001
-	#
+vendor: composer.lock
+	docker-compose exec composer composer install
+
+##
+##Tests
+##-----
+##
+
+coverage: vendor ## Run coverage unit tests
+	docker-compose exec php vendor/bin/phpunit --coverage-html --coverage-text
+
+test: tu tf ## Run unit and functional tests
+
+tu: vendor ## Run unit tests
+	docker-compose exec php vendor/bin/simple-phpunit --exclude-group functional
```

Comment:
simple-phpunit or phpunit ?

Reply:
i thk they are only in project. symfony ask now for simple phpunit

Reply:
resolve with https://github.com/DarkmiraTour/community-event-manager/pull/91/commits/7da60dfc4440b90267e724471b16b596d620952a